docs/administration-guide/10-single-sign-on.md

```diff
@@ -20,34 +18,47 @@ Now existing Metabase users signed into a Google account that matches their Meta
+### LDAP user schema
 
-![Attributes](./images/ldap-attributes.png)
+The **User Schema** section on this same page is where you can adjust settings related to where and how Metabase looks at your LDAP server to authenticate users.
+
+Let's stick with our WidgetCo example from above. If  entries for employees are all stored within an organizational unit in your LDAP server named `People`, you'll want to set the **User search base** field to `ou=People,dc=widgetco,dc=com`. This tells Metabase to begin searching for matching entries at that location within the LDAP server.
+
+While the grayed-out default **User filter** value works for most LDAP servers, this is where you can set a different command for how Metabase finds and authenticates an LDAP entry upon a person logging in.
+
+If you're running [Metabase Pro or Enterprise Edition](https://www.metabase.com/pricing) and using [data sandboxes](../enterprise-guide/data-sandboxes.md), you can utilize existing LDAP [user attributes](../enterprise-guide/data-sandboxes.html#getting-user-attributes) when granting sandboxed access.
+
+### Group mapping
+
+Manually assigning people to [groups](05-setting-permissions.html#groups) in Metabase after they've logged in via SSO can get tedious. Instead, you can take advantage of the groups that already exist in your LDAP directory by enabling [group mappings](/learn/permissions/ldap-auth-access-control.html#group-management). 
+
+Scroll to **Group Schema** on the same LDAP settings page, and click the toggle to enable group mapping. Selecting **Edit Mapping** will bring up a modal where you can create and edit mappings, specifying which LDAP group corresponds to which user group in your Metabase instance.
+
+As you can see below, if you have an **Accounting** group in both your LDAP server and Metabase instance, you'll just need to supply the Distinguished Name from your LDAP server (in the example, it's `cn=Accounting,ou=Groups,dc=widgetco,dc=com`) and select its match from the dropdown of your existing Metabase groups.
 
-Your LDAP directory must have the email field populated or Metabase won't be able to create or log in the user. If
-either name field is missing, Metabase will use a default of "Unknown", and the name can be changed manually in the
-user's account settings.
+![Group Mapping](images/ldap-group-mapping.png)
 
-If you have user groups in Metabase you are using to control access, it is often tedious to have to manually assign a user to a group after they're logged in via SSO. You can take advantage of the groups your LDAP directory uses by enabling Group Mappings, and specifying which LDAP group corresponds to which user group on your Metabase server.
+Note that updates to group membership based on LDAP mappings will only take effect once a person has logged into Metabase again after the update.
```

Comment:
A further clarification here: we'll only update the groups for the user who logs in, not the groups for everyone. 

(Basically, we only fetch data from the LDAP server when absolutely necessary. We can't ever do a full group sync, since we only create users in Metabase once they've logged in the first time.)